localization/english/wc_buildings_l_english.yml

```diff
@@ -118,6 +118,12 @@
  building_thoradins_wall_01_desc:1 "Once a wonder for our people, a testament to our power and our technological prowess, Thoradin's Wall now only stands as a symbol of our downfall. Some parts of the wall still remain and help keep the peace, but the mostly ruined wall is but a shell of the once great shield against those who would wish harm upon us."
  building_thoradins_wall_02:1 "Thoradin's Wall"
  building_thoradins_wall_02_desc:2 "In its full glory, Thoradin's Wall stands proudly as a bastion for our people. We are protected by the towering walls and the brave soldiers guarding us against the dangers outside."
+ building_type_greymane_wall_01:0 "Greymane Wall"
+ building_type_greymane_wall_01_desc:1 "A colossal wall, that shields the denizens of its holder's domain. All unwanted guests must first face this almost insurmountable obstacle."
+ building_greymane_wall_01:1 "Shattered Greymane Wall"
+ building_greymane_wall_01_desc:1 "Once built to protect and isolate the people of Gilneas, the wall has now been shattered, exposing them all."
+ building_greymane_wall_02:1 "Greymane Wall"
+ building_greymane_wall_02_desc:2 "The Greymane Wall was erected by order of King Genn Greymane, as the kingdom of Gilneas isolated itself from the Alliance and the rest of the world due to a disagreement surrounding the internment camps for the orcs, and other disputes."
```

Comment:
I would get rid of the king's name, as it could be built by someone else if he dies/loses the title before the scourge. Mentioning the Alliance and orc encampments is also something that might not have happened at all. Maybe keep it closer to the first description?

Reply:
We should keep the name "Greymane" yes.
If the situation ever changes, we just have to say that it is a sort of tribute to an ancient dynasty.